src/library_sdl.js

```diff
@@ -553,7 +553,7 @@ var LibrarySDL = {
     receiveEvent(event) {
       function unpressAllPressedKeys() {
         // Un-press all pressed keys: TODO
-        for (var keyCode of SDL.keyboardMap) {
+        for (var keyCode of Object.values(SDL.keyboardMap)) {
```

Comment:
Is this the crash fix?

Reply:
Yup, I see this crash when running the interactive tests.